Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2050,6 +2120,64 @@ def pin_message():
         return_error(f"{slack_error}")
 
 
+def fetch_channels():
```

Comment:
I am not sure that it makes sense that we are not using the same core function to fetch channels names. seems to me like fetch_channels_iterable and fetch_channels should call the same function, and the update of the context should be outside.

Reply:
this should probably be merged with get_conversation_by_name as an internal function that will be called to search all or to get one channel name.